src/directive.js

```diff
@@ -26,9 +26,30 @@ export default {
       core.inputHandler(e, el)
     }
 
+    const compositionHandler = (e) => {
+      if (e.target !== el) {
+        return
+      }
+
+      if (e.type === 'compositionstart' || e.type === 'compositionupdate') {
```

Comment:
```suggestion
      if (['compositionstart', 'compositionupdate'].includes(e.type)) {
```

not too much shorter, but easier to add addition checks later.